configs/pontoon.toml

```diff
@@ -327,3 +327,9 @@ locales = [
 [[paths]]
     reference = "en/products/vpn/platform-post-download.ftl"
     l10n = "{locale}/products/vpn/platform-post-download.ftl"
+[[paths]]
+    reference = "en/mozorg/try-picture-in-picture.ftl.ftl"
```

Comment:
Notice this wasn't showing up in Pontoon, just realized there is a double `.ftl` at the end.

Reply:
I will file an issue on this - more issue than I realized, :).